src/test/resources/data/dms_data.json

```diff
@@ -1,3 +1 @@
-{
-  "data": "{\"data\":{\"offender_id\":127,\"offender_name_seq\":254000,\"birth_date\":\"1932-10-26\",\"create_date\":\"2015-05-19\",\"modify_datetime\":\"2022-09-01T18:21:36.915241Z\",\"age\":89,\"create_datetime\":\"2022-09-01T18:21:36.915241Z\",\"audit_timestamp\":\"2022-09-01T18:21:36.915241Z\"},\"metadata\":{\"timestamp\":\"2023-03-12T18:48:45.172406Z\",\"record-type\":\"data\",\"operation\":\"load\",\"partition-key-type\":\"primary-key\",\"schema-name\":\"public\",\"table-name\":\"offenders\"}}"
-}
+{"data": "{\"data\":{\"offender_id\":127,\"offender_name_seq\":254000,\"birth_date\":\"1932-10-26\",\"create_date\":\"2015-05-19\",\"modify_datetime\":\"2022-09-01T18:21:36.915241Z\",\"age\":89,\"create_datetime\":\"2022-09-01T18:21:36.915241Z\",\"audit_timestamp\":\"2022-09-01T18:21:36.915241Z\"},\"metadata\":{\"timestamp\":\"2023-03-12T18:48:45.172406Z\",\"record-type\":\"data\",\"operation\":\"load\",\"partition-key-type\":\"primary-key\",\"schema-name\":\"public\",\"table-name\":\"offenders\"}}"}
```

Comment:
Please revert this change so that we keep the original format. The tests should still pass with the multiline format (which is enforced by a PR check). I fixed this a few weeks back.

Reply:
I created this branch before your changes gets pushed to master/main. I believe the merge to master/main will resolve the conflicts.

Reply:
The version of the code in master contains the fix.